builtin/logical/transit/path_config.go

```diff
@@ -193,19 +193,19 @@ func (b *backend) pathConfigWrite(ctx context.Context, req *logical.Request, d *
 		}
 	}
 
-	autoRotateIntervalRaw, ok, err := d.GetOkErr("auto_rotate_interval")
+	autoRotatePeriodRaw, ok, err := d.GetOkErr("auto_rotate_period")
```

Comment:
Possibly outside the scope of this PR, but should we handle the case where err == nil and ok is false? See https://github.com/hashicorp/vault/blob/main/sdk/framework/field_data.go#L218

Or are we okay with just ignoring a provided nil value of "auto_rotate_period" and keep on going?

Reply:
In this case we're ok continuing with a nil value as the field is not required for a config update. We only need to act when a valid value is provided by updating and persisting the new, provided value.